### PR TITLE
Update node configurations to match changes to test constants

### PR DESCRIPTION
**Pull request by rnicoll**
Updates node configurations to match changes introduced in #875

### CODE REVIEW

**Comment by andr3ej** [comment]
Did you run the demos to check this doesn't introduce any new brokenness?

**Comment by rnicoll** [comment]
Attachment at least doesn't work in current `master` - I've add a commit that fixes the notary lookup there. Re-testing the others too.

**Comment by rnicoll** [comment]
And SIMM valuation demo is broken in master too

**Comment by rnicoll** [comment]
Rebased to resolve merge conflict. All of the affected samples are now tested and existing issues fixed, ready for re-review.

File: samples/simm-valuation-demo/build.gradle [hunk 1]
```diff
@@ -32,8 +32,12 @@ dependencies {
     cordaCompile project(path: ":node:capsule", configuration: 'runtimeArtifacts')
     cordaCompile project(path: ":webserver:webcapsule", configuration: 'runtimeArtifacts')
     cordaCompile project(':core')
-    cordaCompile project(':webserver')
     cordaCompile project(':finance')
+    cordaCompile project(':webserver')
+    cordaCompile project(':test-utils')
```
Comment (Clintonio) [inline samples/simm-valuation-demo/build.gradle]: Again; is test utils required here? It isn't in the node so will not be available at runtime.

**Pull request merged**